backend/app/controllers/MenteesController.php

```diff
@@ -38,6 +42,101 @@ public function singleMentee($id) {
         ]);
     }
 
+    public function login()
+    {
+
+        $credentials = request()->get(['email', 'password']);
+
+        if(!$credentials['email'] || !$credentials['password']) {
+            response()->exit([
+                'error' => '404',
+                'message' => "One of required fields isn't filled.",
+            ]);
+        }
+        $user = User::where('email', $credentials['email'])->first();
+
+        $user = auth()->login($credentials);
+
+        if (!$user) {
+            response()->exit(auth()->errors());
+        }
+
+        response()->json($user);
+    }
+
+    public function menteeInfo()
+    {
+        if (auth()->user()) {
+            $id = auth()->id();
+
+            $menteeInfo = db()
+            ->select("users
+            LEFT JOIN users_info
+            ON users.id = users_info.users_id
+            WHERE id = ?")
+            ->hidden("password")
+            ->bind($id)
+            ->fetchAssoc();
+
+            response()->json([
+                "menteeinfo" => $menteeInfo,
+            ]);
+          } else {
+            response()->json([
+                "error" => 404,
+                "message" => "User isn't logged in.",
+            ]);
+          }
```

Comment:
This will be removed when https://github.com/AnimaVillis/trainingdiary/pull/22/files#r1207756340